build.gradle

```diff
@@ -28,7 +28,7 @@ buildscript {
 plugins {
 	id "com.github.johnrengelman.shadow" version "4.0.2"
 	id 'org.asciidoctor.convert' version '1.5.11'
-	id "me.champeau.gradle.japicmp" version "0.2.6"
+	id "me.champeau.gradle.japicmp" version "0.2.8"
```

Comment:
IIRC we're stuck in 0.2.6 because newest japicmp by default reject introducing new `default method()`s (which we need in 3.4), and the gradle plugin doesn't let us disable that feature.

did you have a strong need for 0.2.8 ?

Reply:
Ah, okay. It worked fine so I proceeded. Will revert